sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ece-request/_tests_/ece-process-integration.scheduler.e2e-spec.ts

```diff
@@ -55,76 +68,93 @@ describe(describeProcessorRootTest(QueueNames.ECEProcessIntegration), () => {
     );
   });
 
-  it("Should process an ECE request file and when there are valid disbursements and applications.", async () => {
-    // Arrange
-    // Student with valid SIN.
-    const student = await saveFakeStudent(db.dataSource);
-    // Student application eligible for e-Cert.
-    const application = await saveFakeApplicationDisbursements(
-      db.dataSource,
-      {
-        student,
-        firstDisbursementValues: [
-          createFakeDisbursementValue(
-            DisbursementValueType.CanadaLoan,
-            "CSLP",
-            1122,
-          ),
-        ],
-        institutionLocation: createFakeInstitutionLocation(undefined, {
-          initialValue: { hasIntegration: true },
-        }),
-      },
-      {
-        offeringIntensity: OfferingIntensity.fullTime,
-        applicationStatus: ApplicationStatus.Enrolment,
-        firstDisbursementInitialValues: {
-          coeStatus: COEStatus.required,
+  it(
+    "Should process an ECE request file and when there are valid disbursements and applications" +
```

Comment:
The "and" makes the sentence harder to read. Was it added on purpose?

Reply:
that was a mistake 😊